python/tvm/relay/grammar/Relay.g4

```diff
@@ -83,8 +83,9 @@ expr
 
   | ident                                     # identExpr
   | scalar                                    # scalarExpr
-  // | expr '.' INT                              # project
-  // | 'debug'                                   # debug
+  | LOCAL_VAR '=' expr ';' expr               # graphExpr
```

Comment:
```suggestion
  | var '=' expr ';' expr               # seq
```

LOCAL_VAR/GRAPH_VAR should be enforced in `_parser.py`. Also I think this case should be added to `visitSeq`, since it shares a lot of code with the other sequencing cases.